test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Import/ImportTests.cs

```diff
@@ -145,6 +146,103 @@ public async Task GivenIncrementalLoad_MultipleInputVersions_ResourceNotExisting
             Assert.Equal(GetLastUpdated("2002"), result.Resource.Meta.LastUpdated);
         }
 
+        [Fact]
+        public async Task GivenIncrementalImportInvalidResource_ThenErrorLogsShouldBeOutput_FailedRCountShouldMatch()
```

Comment:
Nit: Is the R in the test title a typo or suppose to be the work Resource? Also, the test name should be in Given/When/Then format.

Reply:
@LTA-Thinking  Yes, type. Changed to GivenIncrementalImportInvalidResource__WhenImportData_ThenErrorLogsShouldBeOutputAndFailedCountShouldMatch.